app/controllers/admin/api/cms/templates_controller.rb

```diff
@@ -163,4 +164,8 @@ def can_destroy
   def cms_templates
     current_account.templates.but(CMS::EmailTemplate, CMS::Builtin::LegalTerm).order(:id)
   end
+
+  def content?
+    params[:content] == "true"
```

Comment:
Depending on how permissive we want to be here, we could use
```suggestion
    ActiveModel::Type::Boolean.new.cast(params[:content])
```
Of course, this would threat `?full=some_random_string` as true.

Reply:
I considered that but since we don't want to send extra data unless explicitly requested I thought that it'd be better to return the whole data only when the parameter is exactly `true`. Note that `'true'` also evaluates as `false` in my approach.

Reply:
> Note that `'true'` also evaluates as false in my approach.

Do you mean like literally adding quotes in the params? e.g. `?content=true` vs `?content='true'`, where only the first one will return content?

Reply:
> Do you mean like literally adding quotes in the params? e.g. ?content=true vs ?content='true', where only the first one will return content?

Yes